geohash.go

```diff
@@ -127,6 +127,12 @@ func Decode(hash string) (lat, lng float64) {
 	return box.Round()
 }
 
+// DecodeCenter the string geohash to a (lat, lng) point.
```

Comment:
Please could we adjust this comment to differentiate it from the `Decode` function? Perhaps:

```
// DecodeCenter decodes the string geohash to the central point of the bounding box.
```